pkg/easiencryption/base_64.go

```diff
@@ -0,0 +1,52 @@
+package easiencryption
```

Comment:
This package (and folder) should probably be named `easiencoding` - base64 doesn't provide any encryption, it's just a different way to represent data.

Reply:
Nit: I also prefer shorter package names, especially when they're paired with the import statements:

i.e.
```go
github.com/cmsgov/easi-app/pkg/easi_encoding
```

vs

```go
github.com/cmsgov/easi-app/pkg/encoding
```

Reply:
I actually started with this package called `encoding`, but I dislike it because it is already a native go package. The could both be named that, but then we have to look out for naming conflicts. I'd rather name it differently to avoid that. 

I also like how `easi_encoding` looks, but it goes against `go` [recommendations for package naming](https://go.dev/blog/package-names) (no underscores).

I'll change to `easiencoding` for now. If you have a strong preference, I can change it to `encoding`